### PR TITLE
refactor:  update the sveltekit embed recipes

### PR DESCRIPTION
**Pull request by ugurkorkmaz**
Hi, I have updated the Sveltekit Embed Recipes. Could you please review my PR?
Issue(#1399 )

### CODE REVIEW

File: sveltekit-embed/bin/exec.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
+package main
```
Comment (gaby) [inline sveltekit-embed/bin/exec.go:1]: This file should be a Makefile 😂

File: sveltekit-embed/go.mod [hunk 2]
```diff
@@ -0,0 +1,26 @@
+module app
+
+go 1.19
+
+require (
+	github.com/fatih/color v1.15.0
+	github.com/gofiber/fiber/v2 v2.47.0
```
Comment (gaby) [inline sveltekit-embed/go.mod:7]: Fiber v2.48.0 is the current latest

**Review by gaby — CHANGES_REQUESTED** [review]

File: sveltekit-embed/go.mod (hunk 2 above)
Comment (ugurkorkmaz) [reply sveltekit-embed/go.mod]: I updated it.

**Review by gaby — APPROVED** [review]
LGTM

**Review by efectn — APPROVED** [review]

File: .github/dependabot.yml [hunk 3]
```diff
@@ -290,6 +290,14 @@ updates:
     directory: "/entgo-sveltekit"
     schedule:
       interval: "daily"
+  - package-ecosystem: "npm"
+    directory: "/entgo-sveltekit/template"
```
Comment (ReneWerner87) [inline .github/dependabot.yml:294]: wrong directory

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**